### PR TITLE
comments

### PR DESCRIPTION
**Pull request by SilindileShabangu**

### CODE REVIEW

**Review by Lord-Nickelbottoms — APPROVED** [review]
Instead of `element.innerHTML`, use `element.textContent` if you want to change the text of an element.

**Comment by LefaJele-Masemola** [comment]
Please organise your files; maybe have folders.

**Comment by Sisamkele022** [comment]
Add folders and also  I noticed a few minor formatting issues that might improve readability.

**Review by Ayanda-Radebe — DISMISSED** [review]
add an icon and also change the colors

**Review by Sisamkele022 — APPROVED** [review]
looks good

**Review by Sisamkele022 — APPROVED** [review]

File: Profile.css [hunk 1]
```diff
@@ -874,7 +874,7 @@ body {
 
 #chatbot-header { 
 
-    background-color: #07fcbf; 
+    background-color: #27bd97;
```
Comment (Sisamkele022) [inline Profile.css:877]: nice

**Review by Sisamkele022 — APPROVED** [review]

**Review by Ayanda-Radebe — APPROVED** [review]
good changes, looking nice now

**Review by LefaJele-Masemola — APPROVED** [review]

**Pull request merged**